### PR TITLE
Align hidi params with kiota

### LINKED ISSUE #662: Align input parameter names of hidi with kiota commandline tool
As this tool will regularly be used with the kiota tool, it would make sense to align the command line names like input, output, etc.  

@danielmbaluka could you chat with @baywet to align.

### PR DESCRIPTION
**Pull request by MaggieKimani1**
As per [Kiota's documentation](https://microsoft.github.io/kiota/using.html), this PR seeks to align the common parameters between Hidi and Kiota in order to reduce any disparity in the tools.
As of now, these parameters are:
-  `--openapi` in Kiota  vs `--input` in Hidi representing the path to the OpenAPI description/CSDL file used to generate the code files
-  `--output` in Kiota vs `--output` in Hidi representing the output directory path for the generated code files

This PR also seeks to add aliases to our existing parameters for simplicity
Fixes #662

### CODE REVIEW

**Review by danielmbaluka — APPROVED** [review]
LGTM

**Review by baywet — CHANGES_REQUESTED** [review]
hold before you merge please! 
see this comment on other differences https://github.com/microsoft/OpenAPI.NET/issues/662#issuecomment-1020090107

**Comment by MaggieKimani1** [comment]
> hold before you merge please! see this comment on other differences [#662 (comment)](https://github.com/microsoft/OpenAPI.NET/issues/662#issuecomment-1020090107)

@baywet Implemented!

File: src/Microsoft.OpenApi.Hidi/OpenApiService.cs
Comment (baywet) [inline src/Microsoft.OpenApi.Hidi/OpenApiService.cs:182]:
```suggestion
                    using var httpClient = new HttpClient(new HttpClientHandler()
```

and move the client handler to a variable, with a using statement as well

**Review by baywet — CHANGES_REQUESTED** [review]
thanks for taking in the changes, a few comments.

File: src/Microsoft.OpenApi.Hidi/OpenApiService.cs
Comment (MaggieKimani1) [reply src/Microsoft.OpenApi.Hidi/OpenApiService.cs:44]: Yes the goal is to access the stack trace once we log the exceptions.

File: src/Microsoft.OpenApi.Hidi/OpenApiService.cs
Comment (MaggieKimani1) [reply src/Microsoft.OpenApi.Hidi/OpenApiService.cs:182]: Resolving..

File: src/Microsoft.OpenApi.Hidi/OpenApiService.cs
Comment (baywet) [reply src/Microsoft.OpenApi.Hidi/OpenApiService.cs:44]: alright, thanks for the explanation, it makes the code a bit harder to read but I understand the rationale. resovling.

File: src/Microsoft.OpenApi.Hidi/Program.cs
Comment (baywet) [inline src/Microsoft.OpenApi.Hidi/Program.cs:73]:
```suggestion
                await OpenApiService.ProcessOpenApiDocument);
```

**Review by baywet — CHANGES_REQUESTED** [review]

File: src/Microsoft.OpenApi.Hidi/Program.cs
Comment (baywet) [reply src/Microsoft.OpenApi.Hidi/Program.cs:73]: ah ignore my confused self, but please double check the system.commandline API recognizes it's now a tasks and blocks on it, I ran into that issue with kiota a while ago and this leads to very confusing behaviours

Comment (baywet) [reply src/Microsoft.OpenApi.Hidi/Program.cs:73]: Yep sorry, as long as the command invoke blocks on the task we're good

**Review by baywet — APPROVED** [review]

**Pull request merged**
